core/policy_test.go

```diff
@@ -216,7 +216,7 @@ func TestMessageQueuePolicy(t *testing.T) {
 			b.SetTicket([]byte{1})
 			b.SetTimestamp(0) // Tweak if necessary to force CID ordering opposite ticket ordering.
 		})
-		assert.True(t, bytes.Compare(b1.Cid().Bytes(), b2.Cid().Bytes()) > 0)
+		assert.True(t, bytes.Compare(b1.Cid().Bytes(), b2.Cid().Bytes()) != 0)
```

Comment:
Blocking: the intent of this check is to ensure we exercise a pair of blocks where cid and ticket order are inverted to prevent regression.  If the cid orders have flipped without the nonce then we need to tweak the blocks to give us back this property.

cc @anorth.